### ISSUE TITLE
[Bug] Docker Install Method - KeyError: 'MaxLogSize'

### ISSUE THREAD

**Issue by skydivercactus** [DuckBoss/JJMumbleBot#289]
<h2>Please have a look at the <a href="https://github.com/DuckBoss/JJMumbleBot/wiki/F.A.Q.">F.A.Q.</a> for solutions to common problems before posting an issue!</h2>

**Describe the bug**
Bot is not running due to Keyerror for [P_LOG_SIZE_MAX] in a docker install method. 

**To Reproduce**
 sudo docker-compose run bot -ip 'IP' -port 'PORT' -username 'Username' -superuser 'Username'

**Expected behavior**
Bot running

**Screenshots**
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/app/__main__.py", line 286, in <module>
    service.BotService(args.server_ip, int(args.server_port), args.server_password)
  File "/app/JJMumbleBot/core/bot_service.py", line 35, in __init__
    BotServiceHelper.initialize_settings()
  File "/app/JJMumbleBot/lib/helpers/bot_service_helper.py", line 39, in initialize_settings
    runtime_settings.max_log_size = int(global_settings.cfg[C_LOGGING][P_LOG_SIZE_MAX])
  File "/usr/local/lib/python3.9/configparser.py", line 1254, in __getitem__
    raise KeyError(key)
KeyError: 'MaxLogSize'
```

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Version: Latest Docker Version 
 - Python Version:  3.7 

**Additional context**

**Comment by Gymnae** [comment]
My docker install works fine besides from problems with the websocket, which limits the web interface to being able to stop currently playing audio via the button

**Comment by DuckBoss** [comment]
I'll look into this as soon as I can. I will be busy for a couple days.

**Comment by DuckBoss** [comment]
Hi, could you test this branch and let me know if your problem persists? You will have to build/run the docker container.
[Logging Fix Branch](https://github.com/DuckBoss/JJMumbleBot/tree/log_config_fix)
[Patch Notes For Branch](https://github.com/DuckBoss/JJMumbleBot/pull/294)

I've also disabled logging by default since most users probably won't require it. 
To enable logging you can use the `-uselogging` launch parameter for docker builds, or modify the `config.ini` file for non-docker builds.

**Comment by DuckBoss** [comment]
Hello!
The logging fix should be include in the v4.3.1 release, so please try it out and let me know if your problems persist.
